cmake/EthDependencies.cmake

```diff
@@ -41,6 +23,21 @@ set(ETH_SCRIPTS_DIR ${ETH_CMAKE_DIR}/scripts)
 set(Boost_USE_MULTITHREADED ON)
 option(Boost_USE_STATIC_LIBS "Link Boost statically" ON)
 
-find_package(Boost 1.65.0 QUIET REQUIRED COMPONENTS regex filesystem unit_test_framework program_options system)
+set(BOOST_COMPONENTS "regex;filesystem;unit_test_framework;program_options;system")
 
-eth_show_dependency(Boost boost)
+find_package(Boost 1.65.0 QUIET REQUIRED COMPONENTS ${BOOST_COMPONENTS})
+
+# make sure we actually get all required imported targets for boost
```

Comment:
In case the boost version is newer than the cmake version, but the boost version is *less* than 1.70, ``find_package(Boost)`` will *not* define imported targets, so we have to define them ourselves as a workaround...